### DISCUSSION TITLE
Which package installed on Pi?

### DISCUSSION THREAD

**Discussion by TheCeet** [AdguardTeam/AdGuardHome#4994] [Q&A] [answered]
I installed AGH on a Pi 4 with OS DietPI 64-bit.
But where can I see which package I installed?
I'm not sure if I installed ARMv6 or ARMv7

**Comment by ppfeufer** [comment, marked as answer]
```shell
./AdGuardHome -v --version
```
This will throw all the version details in your console, look here for `GOARCH`

You can also run:
```shell
uname -a
```
If it looks something like this:
```plaintext
Linux adguard 5.15.61-v8+ #1579 SMP PREEMPT Fri Aug 26 11:16:44 BST 2022 aarch64 GNU/Linux
```
You can install the `arm64` version. (Note the `aarch64` keyword and the `v8+` addition to the version)
